backend/app/common.py

```diff
@@ -1,8 +1,55 @@
 """
-Miscellaneous utility functions useful throughout the system
+Miscellaneous utility functions and variables useful throughout the system
 """
 from textwrap import dedent
+from app.models import Gender
+from nltk.corpus import stopwords
 
+FEMALE = Gender.objects.all().filter(label="Female")
+MALE = Gender.objects.all().filter(label="Male")
+NONBINARY = Gender.objects.all().filter(label="Neo")
+
+BINARY_GROUP = [FEMALE, MALE]
+TRINARY_GROUP = [FEMALE, MALE, NONBINARY]
```

Comment:
After thinking about this a bit, I think we should remove lines 8 through 13 from this file and instead supply functions that perform the database lookup. That is, something like:
```
def get_gender_female():
  return Gender.objects.all().filter(label="Female")
```

That way we can ensure that the `Gender` model lookup only occurs when needed.

Reply:
I think this makes sense. I've gone back and removed lines 8 through 13.